mm2src/mm2_main/src/lp_swap/taker_swap.rs

```diff
@@ -103,18 +103,22 @@ pub const WATCHER_MESSAGE_SENT_LOG: &str = "Watcher message sent...";
 pub const MAKER_PAYMENT_SPENT_BY_WATCHER_LOG: &str = "Maker payment is spent by the watcher...";
 
 #[cfg(not(target_arch = "wasm32"))]
-pub fn stats_taker_swap_dir(ctx: &MmArc) -> PathBuf { ctx.dbdir().join("SWAPS").join("STATS").join("TAKER") }
+pub fn stats_taker_swap_dir(ctx: &MmArc) -> PathBuf { ctx.global_dir().join("SWAPS").join("STATS").join("TAKER") }
```

Comment:
Do you think we need addresses saved in stats db? We have the taker and maker pubkey but these are mm2 internal ones. If we do, it's not required in this PR or as part of this larger DB refactor task anyway and should be a separate issue.

Reply:
don't have an idea about what they could be used for right now (other than identification).

Reply:
Ok, let's leave it like that for now, and see if anyone will request it or not.